### PR TITLE
Add missing AV link

### PR DESCRIPTION
**Pull request by get-itips**
@altsou please confirm this is the intended missing link to add

closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/9390

### CODE REVIEW

File: Teams/rooms/update-management.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ There are a few pre-requisites for receiving updates through the Pro Management
 |Logitech Sync Services  |Should be installed and running on the Logitech meeting room devices. Required Sync services will be installed automatically from Windows Updates unless blocked. The full Sync package can also be installed. |
 |Windows OS updates |Should be kept enabled and not redirected to WSUS, nor blocked from a networking perspective. Neither GPO nor MDM policies should be used to manage OS updates. |
 |Microsoft Store updates   |Should be turned off. Managed Services will turn off Store updates if found on. |
-|Anti-Virus Software |If you are running AV software on these devices, you should ensure that AV has exclusions in place for Teams and Skype dlls. See here for more details. |
+|Anti-Virus Software |If you are running AV software on these devices, you should ensure that AV has exclusions in place for Teams and Skype dlls. See [here](/microsoftteams/troubleshoot/teams-administration/include-exclude-teams-from-antivirus-dlp) for more details. |
```
Comment (JohanFreelancer9) [inline Teams/rooms/update-management.md:53]:
```suggestion
|Anti-Virus Software |If you are running AV software on these devices, you should ensure that AV has exclusions in place for Teams and Skype dlls. See [How to include or exclude Teams from antivirus or DLP applications](/microsoftteams/troubleshoot/teams-administration/include-exclude-teams-from-antivirus-dlp) for more details. |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@altsou PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by get-itips** [comment]
@tiburd Can you help me getting the right person to review? Thanks

**Comment by altsou** [comment]
Reviewing with the team to verify link. It seems that link was never present, so I'm making sure the suggested link is correct.

**Comment by altsou** [comment]
Link is correct.

**Comment by altsou** [comment]
@scanum Not sure I can merge myself. Verified that link is correct. Are you able to merge to main?

**Comment by scanum** [comment]
Hi @tiburd Hope all is well, is this something you can help with? Thanks!

**Pull request merged**